app/src/js/modules/liveeditor.js

```diff
@@ -20,12 +20,20 @@
      */
     var liveEditor = {};
 
-    var editableTypes = [
+    var contentEditableTypes = [
         'text',
         'html',
         'textarea'
     ];
 
+    // Content types that won't autoreload on changes in modal
```

Comment:
Ideally all fields would emit the `keyup` and `change` events, but some doesn't. These fields won't tell if the the values are altered, so instead we act as they're changed every time they're opened in a modal.